TrueCraft.Launcher/Views/OptionView.cs

```diff
@@ -169,7 +169,7 @@ void OfficialAssetsButton_Clicked(object sender, EventArgs e)
                             stream.Dispose();
                             var jar = ZipFile.Read(ms);
                             var zip = new ZipFile();
-                            zip.AddEntry("pack.txt", "Minecraft textures");
+                            zip.AddEntry("pack.txt", "The default look of Minecraft");
```

Comment:
No, just to reduce ambiguity between this phrase and the default TrueCraft texture pack.